app/templates/main/mapview.html

```diff
@@ -0,0 +1,60 @@
+{% extends 'layouts/base.html' %}
+{% block content %}
+
+
+  <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.6.4/jquery.min.js"></script>
+  <script type="text/javascript" src="https://maps.google.com/maps/api/js?sensor=true"></script>
+  <script type="text/javascript" src="http://hpneo.github.io/gmaps/gmaps.js"></script>
+  <style type="text/css">
+
+    #map {
+      display: block;
+      width: 70%;
+      height: 500px;
+      -moz-box-shadow: 0px 5px 20px #ccc;
+      -webkit-box-shadow: 0px 5px 20px #ccc;
+      box-shadow: 0px 5px 20px #ccc;
+    }
+
+    .ui.button{
+      text-align: center;
+      margin: 0 auto;
+      }
+
+  </style>
+<div class="ui centered grid">
+    <div class=" fourteen wide column">
+        <div class="centered row">
+            <h1>User Map</h1>
+        </div>
+        <div class="centered row">
+            <button class="ui left attached button" >Users</button>
+            <button class="right attached ui button">Resources</button>
+        </div>
+        <div class="centered row">
+            <div id = "map">
+                <script>
```

Comment:
this script tag shouldn't be in the middle of a div html tag. It should be either up above by the other script tags or below after the html that will be displayed.